### ISSUE TITLE
Windows 11 does not seem to work

### ISSUE THREAD

**Issue by JacobGood1** [Tensegritics/ClojureDart#114]
followed the instructions "How to run a sample project" in the readme.

After running ` clj -Mcljd flutter` I get a failure with exit code 65, the failure is:

Cannot run an executable in a subdirectory of a dependency.

**Comment by dupuchba** [comment]
Thx @JacobGood1 - is it okay if we continue this conversation on #115 ?

**Comment by JacobGood1** [comment]
Of course, it is. You do not even need to ask, this is your project after all. 😄
